src/tests/ftest/util/test_utils_pool.py

```diff
@@ -554,6 +554,30 @@ def get_pool_daos_space(self):
         keys = ("s_total", "s_free")
         return {key: getattr(self.info.pi_space.ps_space, key) for key in keys}
 
+    def get_pool_free_space(self, device="scm"):
+        """
+        Method Description
+             Get SCM or NVME free space
+        Keyword Arguments:
+            device {str} -- (default: {"scm"} or "nvme")
+
+        Returns:
+            free_space : Free SCM or NVME space "str"
+        """
+        free_space = 0
+        dev = device.lower()
+        if dev == "scm":
+            gindex = 0
+        else:
+            gindex = 1
+        daos_space = self.get_pool_daos_space()
```

Comment:
We have only two option so not sure if we directly return the value based on SCM/NVMe and remove this many check?

Reply:
Do you mean pass 0 or 1 to the method? 0: scm, 1,:nvme.. That will remove the if condition.

Reply:
Sure. Will do...

Reply:
Done.